### PR TITLE
Update charter to reflect trademark ownership

### PR DESCRIPTION
**Pull request by craigbox**
Our charter refers to "advising the Open Usage Commons on issues relating 
to the Istio trademark and logo".

Those are now the responsibility of the CNCF, after our submission, so I have updated the charter to match.

Technically this is a change to the charter, albeit a correction, so I'll seek 80% vote from Steering by 👍 on this PR.

### CODE REVIEW

**Comment by kfaseela** [comment]
LGTM

**Comment by kfaseela** [comment]
Will wait for one more review to be merged

**Review by irisdingbj — COMMENTED** [review]
LGTM， Thanks

**Review by ctrath — APPROVED** [review]

**Comment by craigbox** [comment]
ping @cetezadi @thisisnotapril @hzxuzhonghu for the final countdown

**Comment by justinpettit** [comment]
LGTM

**Review by hzxuzhonghu — APPROVED** [review]

**Comment by craigbox** [comment]
Vote passes. Thanks!

**Pull request merged**
